src/defaultOptions.js

```diff
@@ -130,6 +130,15 @@ module.exports = {
       backgroundPosition: 'center',
       backgroundRepeat: 'no-repeat',
     },
+    indeterminateIcon: iconColor => `<svg viewBox="0 0 16 16" fill="${iconColor}" xmlns="http://www.w3.org/2000/svg"><rect width="8" height="2" x="4" y="7" rx="1" /></svg>`,
+    indeterminateIconColor: defaultTheme.colors.white,
+    '&:indeterminate': {
```

Comment:
One detail here is to also define a `:focus:indeterminate` selector. The `borderColor` of this `:indeterminate` overwrites the `borderColor` of `:focus`.

```suggestion
    '&:focus:indeterminate': {
      borderColor: defaultTheme.colors.blue[400]
    },
    '&:indeterminate': {
```